test/helpers/getDescriptorFromProvider.js

```diff
@@ -69,7 +69,9 @@ describe('getDescriptorFromProvider', () => {
     const projectRoot = { root: '' };
     const getProjectExtras = require('../../src/helpers/getProjectExtras');
 
-    getDescriptorFromProvider(provider, { Components, projectConfig, projectRoot, getExtras });
```

Comment:
let's revert this change; otherwise since the entire object literal isn't on one line, each item would have to go on its own line.